shell/platform/darwin/macos/framework/Source/FlutterExternalTextureMetal.h

```diff
@@ -27,4 +27,15 @@
  */
 - (BOOL)populateTexture:(nonnull FlutterMetalExternalTexture*)metalTexture;
 
+/**
+ * Create FlutterMetalExternalTexture from YUV pixelbuffer.
+ */
+- (BOOL)populateTextureFromYUVAPixelBuffer:(nonnull CVPixelBufferRef)pixelBuffer
+                                textureOut:(nonnull FlutterMetalExternalTexture*)textureOut;
+/**
+ * Create FlutterMetalExternalTexture from RGBA pixelbuffer.
+ */
+- (BOOL)populateTextureFromRGBAPixelBuffer:(nonnull CVPixelBufferRef)pixelBuffer
+                                textureOut:(nonnull FlutterMetalExternalTexture*)textureOut;
```

Comment:
You are repeating `texture` here, and making them inconsistent with the existing `populateTexture:`. If we do want this method structure (which I'm still not sold on), the names should be `poplulateTexture:fromRGBAPixelBuffer:` and similarly for the other version.

Reply:
> You are repeating `texture` here, and making them inconsistent with the existing `populateTexture:`. If we do want this method structure (which I'm still not sold on), the names should be `poplulateTexture:fromRGBAPixelBuffer:` and similarly for the other version.

Thank @stuartmorgan
You are right.
It can be satisfied if it is a local function, and there is no need to expose the interface to the outside.